styles.css

```diff
@@ -0,0 +1,186 @@
+  font-family: Roboto;
+  src: url(fonts\Roboto-Regular.ttf);
+}
+
+h1 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+html body {
+  margin: 0;
+  padding: 0;
+  overflow: unset;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.greetings {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: black;
+  text-align: center;
+}
+
+#topbar {
+  width: 375px;
+  height: 92px;
+  display: flex;
+  flex-direction: column;
+}
+
+#appbar {
+  width: 100%;
+  height: 44px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#timer-cont {
+  display: flex;
+  flex-direction: column;
+  margin-left: 21px;
+  width: 54px;
+  height: 21px;
+  justify-content: center;
+}
+
+#timer-text {
+  width: 100%;
+  height: 16px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+}
+
+#statusbar {
+  width: 66.66px;
+  height: 11.34px;
+  margin-right: 17.67px;
+  background-image: url("/images/statusbar.png");
+}
+
+#toolbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 48px;
+}
+
+#logo-placeholder {
+  margin-left: 12px;
+  width: 24px;
+  height: 48px;
+}
+
+#hamburger-icon {
+  width: 18px;
+  height: 12px;
+  background-image: url("images/hamburger_Icon.png");
+}
+
+#hamburger-cont {
+  display: flex;
+  margin-right: 30px;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+}
+
+#home {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#home-bg-img {
+  width: 375px;
+  height: 653.32px;
+  overflow: hidden;
+  background-image: url("images/header-illsutration-mobile.png");
+  z-index: -1;
+  margin-top: -494px;
+}
+
+.black-text {
+  color: black;
+}
+
+.green-text {
+  color: #36b37e;
+}
+
+#greeting-container {
+  margin-top: 54px;
+  width: 327px;
+  height: 211px;
+}
+
+#message-cont {
+  margin-top: 15px;
+  width: 327px;
+  height: 164px;
+  overflow-wrap: break-word;
+  text-align: center;
+}
+
+#support-txt {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+}
+
+#sm-icons-cont {
+  margin-top: 22.89px;
+  width: 170.17px;
+  height: 19.62px;
+  display: flex;
+  justify-content: space-around;
+}
+
+#twitter {
+  background-image: url("images/sm-icons/twitter.png");
+  width: 19.99px;
+  height: 19.58px;
+}
+
+#linkedin {
+  background-image: url("images/sm-icons/linkedin.svg");
+  width: 20px;
+  height: 19.62px;
+}
+
+#medium {
+  background-image: url("images/sm-icons/medium.png");
+  width: 20px;
+  height: 15.69px;
+}
+
+#github {
+  background-image: url("images/sm-icons/github.png");
+  width: 17.99px;
+  height: 19.62px;
+}
+
+#angellist {
+  background-image: url("images/sm-icons/angellist.png");
+  width: 14.17px;
+  height: 19.62px;
+}
```

Comment:
Kindly implement the following design changes :

- [ ] Please the introductory text font color is not in the right color of `#172B4D`, kindly change it from the black color to the one indicated in the design which I listed
- [ ]  please also change the font colour of the paragraph `I can help you build a product feature...` to the color i mentioned above which is the same color in the Figma template 😄 

Your design :
![image](https://github.com/s2jack/create-portfolio-website1/assets/20537712/ba78d454-4bec-4965-b127-3887223e3b59)

Figma design :
![image](https://github.com/s2jack/create-portfolio-website1/assets/20537712/056fd945-c00e-4176-810a-2cfeb56ed528)